db/glue.c

```diff
@@ -392,20 +392,14 @@ int trans_start_set_retries(struct ireq *iq, tran_type *parent_trans,
     return rc;
 }
 
-tran_type *trans_start_socksql(struct ireq *iq, int ignore_newer_updates,
-                               int trak)
+tran_type *trans_start_socksql(struct ireq *iq, int trak)
 {
     void *bdb_handle = bdb_handle_from_ireq(iq);
     tran_type *out_trans = NULL;
     int bdberr = 0;
 
     iq->gluewhere = "bdb_tran_begin_socksql";
-    if (gbl_extended_sql_debug_trace) {
```

Comment:
This trace probably should be kept without the ignore_newer_updates info.  It tracks transaction begins, aborts, commits and so on.